.gitlab/deploy_packages/nix.yml

```diff
@@ -185,7 +185,7 @@ deploy_staging_iot_agent:
   needs: ["build_iot_agent-binary_x64"]
   script:
     - $S3_CP_CMD $S3_ARTIFACTS_URI/iot/agent ./agent
```

Comment:
Here the script can fail on that line. I did the modification below for consistency and will do a follow-up PR for such calls